controller/transfer/labelstudio/project_update_manager.py

```diff
@@ -0,0 +1,190 @@
+import json
+from typing import Dict, Any, List, Tuple
+
+from controller.transfer.labelstudio.util import create_unknown_users
+from controller.transfer.record_transfer_manager import download_file
+from submodules.model import enums, RecordLabelAssociation
+from submodules.model.business_objects import (
+    upload_task,
+    labeling_task,
+    record_label_association,
+    general,
+    labeling_task_label,
+)
+from controller.upload_task import manager as upload_task_manager
+
+
+def manage_data_import(project_id: str, task_id: str) -> None:
+    task = upload_task.get(project_id, task_id)
+    file_path = download_file(project_id, task)
+    mappings = json.loads(task.mappings)
+    user_mapping = mappings.get("users")
+    user_mapping = create_unknown_users(user_mapping)
+    prioritize_existing = mappings.get("prioritizeExisting")
+    tasks_by_labels = __get_tasks_by_label_id(project_id)
+    label_id_lookup = __get_label_id_lookup(project_id)
+    user_ids = __get_user_ids(user_mapping)
+    label_ids = __get_projects_label_ids(project_id)
+
+    with open(file_path) as file:
+        data = json.load(file)
+        association_data = __extract_association_data(
+            data, user_mapping, user_ids, label_id_lookup
+        )
+        record_ids = association_data.keys()
+        record_label_associations, remove_list = __check_existing_associations(
+            project_id,
+            association_data,
+            prioritize_existing,
+            record_ids,
+            user_ids,
+            label_ids,
+            tasks_by_labels,
+        )
+        if remove_list:
+            record_label_association.delete_by_ids(project_id, remove_list)
+
+        record_label_association.create_by_record_user_label_dict(
+            project_id, record_label_associations
+        )
```

Comment:
can we maybe chunk this in 250 record id packs? I'm a bit worried that a transfer of a lot of data will run into issues ->

if I'm not mistaken this part can be run in a for loop with record_ids splitted

```suggestion
        for record_pack in chunk_list(record_ids):
            record_label_associations, remove_list = __check_existing_associations(
                project_id,
                association_data,
                prioritize_existing,
                record_pack,
                user_ids,
                label_ids,
                tasks_by_labels,
            )
            if remove_list:
                record_label_association.delete_by_ids(project_id, remove_list)

            record_label_association.create_by_record_user_label_dict(
                project_id, record_label_associations
            )
```

Reply:
yes, this should work, changed it to chunked check and creation